### PR TITLE
Added ability to save address for contract

### PR DESCRIPTION
**Pull request by KyrylR**
* [Added ability to save address for contract](https://github.com/dl-solarity/hardhat-migrate/commit/8d6f391c8b4ff1b33353cb15b4ad51bbeb587a07)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solarity/hardhat-migrate",
-  "version": "2.0.1",
+  "version": "2.1.1",
```
Comment (Arvolear) [inline package.json:3]: This is not how versioning works

Comment (Arvolear) [reply package.json:3]: Do breaking change anything?

**Comment by Arvolear** [comment]
Suppose tests are for the weak xD

File: package.json (hunk 1 above)
Comment (KyrylR) [reply package.json:3]: Yep, a few core dependencies has been updated

**Review by Arvolear — APPROVED** [review]
LGTM!

**Pull request merged**